### PR TITLE
Log trigger limits

### PR DESCRIPTION
**Pull request by thedriftofwords**
- [Main content](https://documentation-git-fork-thedriftofwords-080-351a16-chainlinklabs.vercel.app/chainlink-automation/concepts/automation-concepts#maximum-logs-processed-for-log-trigger-upkeeps)
- [Cross-linked at top of log trigger guide](https://documentation-git-fork-thedriftofwords-080-351a16-chainlinklabs.vercel.app/chainlink-automation/guides/log-trigger#understanding-maximum-logs-processed)

### CODE REVIEW

File: src/content/chainlink-automation/concepts/automation-concepts.mdx [hunk 1]
```diff
@@ -29,3 +29,21 @@ These are the jobs or tasks that you execute onchain. For example, you can call
...
+
+Expect the following numbers of logs to be processed:
+
+| Chain     | Logs per block                             |
+| --------- | ------------------------------------------ |
+| Ethereum  | 20                                         |
+| BSC(BNB)  | 5                                          |
+| Polygon   | 5                                          |
+| Avalanche | 5                                          |
+| Fantom    | 1                                          |
```
Comment (infiloop2) [inline src/content/chainlink-automation/concepts/automation-concepts.mdx:45]: we don't have log triggers on fantom so can remove that

**Review by infiloop2 — COMMENTED** [review]
looks great! Just need to confirm the Logs per block values on some chains, will sync on slack

File: src/content/chainlink-automation/concepts/automation-concepts.mdx [hunk 2]
```diff
@@ -29,3 +29,21 @@ These are the jobs or tasks that you execute onchain. For example, you can call
...
+Chainlink Automation nodes look back over a limited range of the latest blocks on any particular chain. During this process, the nodes process a limited number of logs per block per upkeep, using a minimum dequeue method to ensure that the latest logs are processed first. After this, the nodes may process additional remaining logs on a best effort basis, but this is not guaranteed. If you need all the remaining logs to be processed, configure a manual trigger as backup.
+
+Expect the following numbers of logs to be processed:
+
+| Chain     | Logs per block                             |
+| --------- | ------------------------------------------ |
+| Ethereum  | 20                                         |
+| BSC(BNB)  | 5                                          |
+| Polygon   | 5                                          |
+| Avalanche | 5                                          |
```
Comment (infiloop2) [inline src/content/chainlink-automation/concepts/automation-concepts.mdx:44]: can we change these to 4 for BSC, Polygon, Avalanche, Optimism, Base

File: src/content/chainlink-automation/concepts/automation-concepts.mdx (hunk 1 above)
Comment (thedriftofwords) [reply src/content/chainlink-automation/concepts/automation-concepts.mdx]: removed

File: src/content/chainlink-automation/concepts/automation-concepts.mdx (hunk 2 above)
Comment (thedriftofwords) [reply src/content/chainlink-automation/concepts/automation-concepts.mdx]: updated

**Review by infiloop2 — APPROVED** [review]

**Review by khadni — APPROVED** [review]

**Pull request merged**